pkg/cli/config_test.go

```diff
@@ -121,6 +121,91 @@ MachineType = "machine-type-bar"
 	}
 }
 
+func TestImportAcloudConfig(t *testing.T) {
+	const acloudConfig = `project: "foo" # project zero
+# Zone comment
+zone: "foo-central1-c" # zone: "invalid"
+# machine_type: "comment"
+machine_type: "foo-standard-4"
+network: "default"
+
+# Cuttlefish host image
+# stable_host_image_family: "acloud-release"
```

Comment:
you need more test cases, for example spaces and tabs in any unexpected place in the line.

Reply:
done